src/runtime.c

```diff
@@ -1,10 +1,17 @@
 // SPDX-License-Identifier: BSD-3-Clause
 /* Copyright 2021-2022, Intel Corporation */
 
+#include <stdlib.h>
+
 #include "libminiasync/runtime.h"
 #include "core/os_thread.h"
 #include "core/os.h"
+#include "core/util.h"
+
+#if defined(__x86_64__) || defined(__amd64__) || defined(_M_X64) \
```

Comment:
This ifdef should probably in the util.h

Reply:
Done.